src/Voron/Data/Tables/Table.cs

```diff
@@ -619,6 +619,32 @@ public IEnumerable<SeekResult> SeekForwardFrom(TableSchema.SchemaIndexDef index,
             }
         }
 
+        public long GetNumberEntriesFor(TableSchema.FixedSizeSchemaIndexDef index, long afterValue, out long totalCount)
+        {
+            var fst = GetFixedSizeTree(index);
+
+            totalCount = fst.NumberOfEntries;
+            if (afterValue == 0 || totalCount == 0)
+                return totalCount;
+
+            long count = 0;
+            using (var it = fst.Iterate())
```

Comment:
That can be extremely expensive, and it will require us to force load all the relevant data.